### DISCUSSION TITLE
v2022-beta.2

### DISCUSSION THREAD

**Discussion by image-et-son** [image-et-son/p600fw#160] [Announcements]
This development stream is based on Version 2.1 RC3 (last commit from GliGli September 6th 2015).

The beta release 2022 (2) is feature complete. Beta testing started on March 27th 2022. The intention is to independently assess the overall stability and handling of the firmware as well as the quality, accessibility, legibility of new and changed functions.

The beta release comes with a User Manual which is part of the release.

For more information and contact see https://prophet600revisited.blogspot.com/

Beta 2022 (2) contains some corrections based on feedback from beta 2022 (1):
- Typos and updates of the User manual
- Fix for modulation delay glitches on new notes
- Documented MIDI specification for tool development
- Fixes in UI for patch managememt mode
- Move of LFO Sync from MIDI CC 58 to MIDI CC 77 (for better compatibility with MIDI tools)
- Change in the relation of pitch and envelope vintage for lower spread values
- Various small UI changes

License
=======

Everything is under GPL v3 license, except files that have their own license in the header.

<hr /><em>This discussion was created from the release <a href='https://github.com/image-et-son/p600fw/releases/tag/v2022-beta.2'>v2022-beta.2</a>.</em>

**Comment by mdee7777** [comment]
Hi I just installed this new version via sysex and immediately was getting stuck notes when triggering the 5th note in a chord. Went back to the previous v2 firmware and the problem went away. So unfortunately I can't use this at the moment.

**Comment by image-et-son** [comment]
Thanks for trying this out.

Did you load this version here or that version: https://github.com/image-et-son/p600fw/releases/tag/v2022-rc1

I'm asking because your first post was on the rc1 and that is also the latest version. Having said this, this shouldn't really happen and it is the first error of that type. Would you take the time to try it out again and tell me if that behaviour is reproducible (always happens after the same sequence of notes) or "random"? And second, if you leave it to ring, does it stay constant or does it fade slowly with the pitch also going down slowly?

Thanks!

**Reply by matrix12x** [reply]
I tried reproducing the error described above re: the hanging note after a 5th note is added to a chord, and could not reproduce it. I also tried a 6th note. I tried using patches with both short and relatively long release (about 75-85) on the Amp EG.
